front/src/applications/operationalStudies/components/MacroEditor/consts.ts

```diff
@@ -40,67 +41,89 @@ export const DEFAULT_TRAINRUN_CATEGORY: TrainrunCategory = {
   sectionHeadway: 0,
 };
 
-export const DEFAULT_TRAINRUN_FREQUENCIES: TrainrunFrequency[] = [
+/**
+ * Return the default TrainrunFrequencies with their translations.
+ */
+export const getDefaultTrainrunFrequencies = (
+  t: TFunction<'operationalStudies/scenario'>
+): TrainrunFrequency[] => [
+  {
+    id: 1,
+    order: 1,
+    frequency: 1440,
+    offset: 0,
+    name: t('macroEditor.uniqueTrainSchedule'),
+    shortName: '-',
+    linePatternRef: '120',
+  },
   {
     id: 2,
-    order: 0,
+    order: 2,
     frequency: 30,
     offset: 0,
-    name: 'Half-hourly',
+    name: t('macroEditor.intervalXmin', { minutes: 30 }),
     shortName: '30',
     linePatternRef: '30',
   },
   {
-    id: 3, // default NGE frequency takes id 3
-    order: 1,
+    id: 3,
+    order: 3,
     frequency: 60,
     offset: 0,
-    name: 'Hourly',
-    /** Short name, needs to be unique */
+    name: t('macroEditor.intervalXmin', { minutes: 60 }),
     shortName: '60',
     linePatternRef: '60',
   },
   {
     id: 4,
-    order: 2,
+    order: 4,
     frequency: 120,
     offset: 0,
-    name: 'Two-hourly',
+    name: t('macroEditor.intervalXmin', { minutes: 120 }),
     shortName: '120',
     linePatternRef: '120',
   },
 ];
 
-export const DEFAULT_TRAINRUN_FREQUENCY: TrainrunFrequency = DEFAULT_TRAINRUN_FREQUENCIES[1];
+export const TRAIN_SCHEDULE_FREQUENCY_ID = 1;
 
-export const DEFAULT_TRAINRUN_TIME_CATEGORY: TrainrunTimeCategory = {
-  id: 0, // In NGE, Trainrun.DEFAULT_TRAINRUN_TIME_CATEGORY
-  order: 0,
-  name: 'Default',
-  shortName: '7/24',
-  dayTimeInterval: [],
-  weekday: [1, 2, 3, 4, 5, 6, 7],
-  linePatternRef: '7/24',
-};
+export const DEFAULT_PACED_TRAIN_FREQUENCY_IDS = [2, 3, 4];
```

Comment:
This is likely to get out-of-date. Would be safer to define static frequencies with the translation key as a the name, map the name with `t()` in `getDefaultTrainrunFrequencies()`, and use here:

```ts
export const PACED_TRAIN_FREQUENCY_IDS = TRAINRUN_FREQUENCIES.filter((freq) freq.id !== TRAIN_SCHEDULE_FREQUENCY_ID).map((freq) => freq.id);
```